microsoft-edge/webview2/release-notes.md

```diff
@@ -34,6 +34,43 @@ To load WebView2, the minimum version of Microsoft Edge or the WebView2 Runtime
 
 To use a prerelease SDK along with a Microsoft Edge preview channel, see [Test upcoming APIs and features](how-to/set-preview-channel.md).
 
+
+<!-- ====================================================================== -->
+
+## 1.0.1245.22
+  
+Release Date: June 15, 2022  
+  
+[NuGet package for WebView2 SDK 1.0.1245.22](https://www.nuget.org/packages/Microsoft.Web.WebView2/1.0.1245.22)  
+  
+For full API compatibility, this version of the WebView2 SDK requires WebView2 Runtime version 102.1245.22 or higher.
+
+### General
+
+#### Promotions
+
+The following items are now stable:
+
+* The [Server Certificate API](/microsoft-edge/webview2/reference/win32/icorewebview2_14?view=webview2-1.0.1245.22&preserve-view=true) which provides an option to trust the server's TLS certificate at the application level. It renders the page without prompting the user about TLS or providing the ability to cancel the web request.
+
+*  The [ClearBrowsingData API](/microsoft-edge/webview2/reference/win32/icorewebview2profile2?view=webview2-1.0.1245.22&preserve-view=true) which allows developers to programmatically clear specific data types for a duration:
+   * `clearBrowsingDataInTimeRange`

```

Comment:
We also have a `ClearBrowsingData` and the API name should be capitalized.

```suggestion
   * `ClearBrowsingData`
   * `ClearBrowsingDataInTimeRange`
```

Reply:
@mikehoffms And the API names here?

Reply:
fixed in https://github.com/MicrosoftDocs/edge-developer/pull/2011/files